src/InformaticsGateway/Properties/launchSettings.json

```diff
@@ -3,7 +3,7 @@
     "Monai.Deploy.InformaticsGateway": {
       "commandName": "Project",
       "environmentVariables": {
-        "DOTNET_ENVIRONMENT": "Development"
+        "DOTNET_ENVIRONMENT": "Local"
```

Comment:
Any difference between using Local vs Development?

Reply:
Just in MWM tests were/are using appsetting.development.json to read settings from (bad Idea!) so making a copy of it and using that for local development only was my solution (as my machine has services running on different ports)
happy to revert this file, if you think thats best